src/js/components/Box/doc.js

```diff
@@ -164,6 +164,7 @@ export const doc = Box => {
     border: PropTypes.oneOfType([
       PropTypes.bool,
       PropTypes.oneOf([
+        'border',
```

Comment:
what does adding a `border` string to the border prop supposed to mean?
```<Box border="border" /> ```

Looks like the type boolean is included in line #165.